panacea/rpc_client.go

```diff
@@ -0,0 +1,113 @@
+package panacea
+
+import (
+	"context"
+	"fmt"
+	ics23 "github.com/confio/ics23/go"
+	"github.com/cosmos/ibc-go/v2/modules/core/23-commitment/types"
```

Comment:
I brought the logic to verify the merkle proof in the ibc module. This works fine, but I'm not sure if I should continue to use it from this module.
First, let's use it as is.